source/changelog.txt

```diff
@@ -12,6 +12,26 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.2.15
+-------
+
+*Released July 30, 2024*
+
+
+New features in this release:
+
+- Added helper for ``shardMoveCollection`` and ``unshardCollection`` commands :issue:`MONGOSH-1848`. 
+  
+Bug fixes in this release:
+
```

Comment:
Before we merge, can you flip the Jira and the description, and turn them into an ordered list? 
- MONGOSH-####: Description
- MONGOSH-####: DEscription

Reply:
Sure, per some of the previous RN's I used a format with `-`. 

For example v2.2.9:

```
- :issue:`MONGOSH-1785` - ``monogosh`` now supports Ubuntu 24.04. 
```